product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Identifying your image names'
+---
+
+For a quick installation with the aim of testing the product, see the [Quickstart](/postgres_distributed_for_kubernetes/latest/quickstart/). 
+For more targeted testing or production purposes, 
+the following section provides an overview of how to select a specific operand and proxy image version that goes in line with your Postgres distribution.
+
+## Operand image name 
+
+See [Operand images](/postgres_distributed_for_kubernetes/latest/private_registries/#operand-images) for general information about the images.  
+
+### Name elements 
+
+The operand image name is composed of three elements: the postgres type, EDB Postgres Distributed, and the build version.
+
+**The Postgres type:** The first step towards identifying your image names is knowing which Postgres type you are going to use, and how it translates to an image name. 
+
+| Postgres type                | Image name            |
+|------------------------------|-----------------------|
+| EDB Postgres Advanced Server | edb-postgres-advanced |
+| EDB Postgres Extended Server | edb-postgres-extended |
+| PostgreSQL                   | postgresql            |
+
+**EDB Postgres Distributed:** The next step is identifying how Postgres Distributed translates to an image name. This is the same for all environment configurations: 
+
+| Product name             | Image name |
+|--------------------------|------------|
+| EDB Postgres Distributed | pgd        |
+
+**Build version:** Last, the build version is 1, unless a new image is created in the scope of a security patch.
+
+### Assembling the operand image name 
+
+These identifiers, together with their versions, translate to an operand image name. Here are a few examples: 
+
+| Postgres + version         | EDB Postgres Distributed + version | Build | Operand image name                       |
+|----------------------------|------------------------------------|-------|------------------------------------------|
+| EDB Postgres Advanced 16.2 | PGD 5.4.1                          | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
+| EDB Postgres Extended 15.6 | PGD 5.4.0                          | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
+| PostgreSQL 16.2            | PGD 5.4.1                          | 1     | postgresql-pgd:16.2-5.4.1-1              |
```

Comment:
What do the + signs mean in the headers?
And, given the example table, is there  a pointer to some place where the user can get a listing?
We devs don't have it. Just putting on my reader hat. If I don't see the version I want here, where do I go?

Reply:
>> What do the + signs mean in the headers?
I think I was trying to make clear that the image name is composed of the product name + version, but now that you point this out, I think it is kind of self-explanatory. If I get rid of the +, it still makes sense. 

```suggestion
| Postgres version         | EDB Postgres Distributed version | Build | Operand image name                       |
|----------------------------|------------------------------------|-------|------------------------------------------|
| EDB Postgres Advanced 16.2 | PGD 5.4.1                          | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
| EDB Postgres Extended 15.6 | PGD 5.4.0                          | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
| PostgreSQL 16.2            | PGD 5.4.1                          | 1     | postgresql-pgd:16.2-5.4.1-1              |
```

>> Is there a pointer to some place where the user can get a listing?
I had the same question. Unfortunately, the listing can only be found on Cloudsmith, and not all customers have access to it 😞 . 
I considered manually adding a list, but that would be really hard to maintain as new versions and combinations are added.